source/cgal/surface_mesh.cc

```diff
@@ -55,7 +55,7 @@ namespace
           Assert(false, ExcInternalError());
           break;
       }
-    if (clockwise_ordering == false)
+    if (clockwise_ordering == true)
       std::reverse(indices.begin(), indices.end());
```

Comment:
Can you isolate this change, and the changes in the existing test file outputs in a commit of its own?

Reply:
Done, thanks.